content/docs/operating/security.md

```diff
@@ -177,6 +182,38 @@ and bandwidth.
 It is recommended to monitor all components for failure, and to have them
 automatically restart on failure.
 
+## Server-side TLS and authentication
+
+In the future, server-side TLS support will be rolled out to the different
+projects under the Prometheus organization. Those projects include Prometheus,
+Alertmanager, Pushgateway and the official exporters.
+
+The golang projects will share the same TLS library, which will be based on the
+Golang vanilla TLS library. We keep [Golang's default TLS
+parameters](https://golang.org/pkg/crypto/tls/#Config), with one exception: we
+support only TLSv1.2 and higher.
+
+Research will be made later on the non-golang exporters to see how to best
+address server-side TLS, but you can expect a policy close to this one.
+
+The TLS support on the projects should enable you to build a secure tunnel
```

Comment:
I'd predicate this on the special needs first, as currently written it makes it sounds like the primary purpose of this is the tunnel rather than this being primarily intended to be used standalone.

Reply:
addressed